lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -326,6 +359,84 @@ static bool sinkCode(Function *F) {
       }
     }
 
+    // Sink Transpose below ChannelShuffle (Reshape->Transpose->Reshape)
```

Comment:
Can you add a bit more of an explanation of what happens when we successfully sink? E.g. mention something about changing the shuffle dimensions of the Transpose